optimize_images/argument_parser.py

```diff
@@ -93,6 +93,15 @@ def get_args():
     parser.add_argument('-jobs', dest="jobs",
                         type=int, default=0, help=jobs_help)
 
+    only_summary_help = 'Show only the summary'
+    parser.add_argument('--only-summary', action='store_true', help=only_summary_help)
+
+    only_progress_help = 'Show only the current progress'
+    parser.add_argument('--only-progress', action='store_true', help=only_progress_help)
+
+    quiet_help = 'Quiet mode, output nothing'
+    parser.add_argument('--quiet', action='store_true', help=quiet_help)
```

Comment:
It would be nice if these new options were returned to main as an `OutputConfiguration` object, instead of 3 separate variables. That long list of return values has over time grown bigger than it should :-)